crates/red_knot_python_semantic/src/types.rs

```diff
@@ -707,6 +726,7 @@ impl<'db> Type<'db> {
             | Type::Tuple(_)
             | Type::LiteralString
             | Type::None => Type::Unknown,
+            Type::Todo => Type::Todo,
```

Comment:
Again, I'd prefer if we keep `Todo` close to `Any` and `Unknown` cases in matches, since they are closely related.